vocab_list/views.py

```diff
@@ -75,6 +75,6 @@ def get_context_data(self, **kwargs):
 
     def form_valid(self, form):
         vl, _ = PersonalVocabularyList.objects.get_or_create(user=self.request.user, lang=form.cleaned_data["lang"])
-        entries = vl.load_tab_delimited(form.cleaned_data["data"], familiarity=form.cleaned_data["rating"])
```

Comment:
not sure why this was needed or why it's in the same context as the change to the `VocabularyList` method above as this goes to the `PersonalVocabularyList` which has a `familiarity` keyword argument.

Reply:
Sorry this took so long to get back to you on this. As I remember, we removed the "familiarity=" because `load_tab_delimited` method on the `PersonalVocabularyList` object is not expect a key-value pair, so it was being turned into key-key-value, i.e., one extra level. This was only a problem when uploading a vocabulary list.